### PR TITLE
[angular][xmcloud] Init CloudSDK on browser

### PR DESCRIPTION
**Pull request by art-alexeyenko**
- [x] This PR follows the [Contribution Guide](https://github.com/Sitecore/jss/blob/dev/CONTRIBUTING.md)
- [x] Changelog updated
- [ ] Upgrade guide entry added

## Description / Motivation
- Initializes CloudSDK on browser for Angular app
- Allows for other parts of app to use events etc

Note: adding events init TBD

## Testing Details
- [ ] Unit Test Added
- [ ] Manual Test/Other (Please elaborate)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/routing/scripts/cloud-sdk-init.component.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { CloudSDK } from '@sitecore-cloudsdk/core/browser';
+import { environment } from '../../../environments/environment';
+import { isServer } from '@sitecore-jss/sitecore-jss/utils';
+
+/**
+ * Component that renders editing scripts and client data for the current page in Sitecore Editor.
```
Comment (yavorsk) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/routing/scripts/cloud-sdk-init.component.ts:7]: This description should be updated.

**Review by yavorsk — CHANGES_REQUESTED** [review]
Looks good! 
See just one thing regarding the component description.
Also - does it make sense to rename the component or the selector for consistency - `cloud-sdk-init` vs `app-cloudsdk-init`?

**Comment by yavorsk** [comment]
Should we add the sitecore-cloudsdk dependencies to angular-xmcloud as well?

**Comment by art-alexeyenko** [comment]
> Should we add the sitecore-cloudsdk dependencies to angular-xmcloud as well?

true - it's now on angular side only

**Review by yavorsk — APPROVED** [review]

**Pull request merged**